Tests/SwiftCliCoreTests/NestedArrayBuilderTest.swift

```diff
@@ -16,3 +16,13 @@ class NestedArrayMakerTest: XCTestCase {
     }
 
 }
+
+class MultipleStitchExpanderTest: XCTestCase {
+
+
+    func testNestedArrayBuilder() throws {
+        let result = NestedArrayBuilder().expandRow(row: ["k4", "p1"])
+        expect(result).to(equal(["k1", "k1", "k1", "k1", "p1"]))
+    }
```

Comment:
Let's test some edge cases! What happens when you input "kx py"?

Reply:
As you said in pairing, let's also cover "k0"!

Reply:
Added these!